src/components/AddInput/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import { inline } from "./style.css";
+
+interface IAddInput {
+  setterClose: React.Dispatch<React.SetStateAction<boolean>>;
+  add: (arg0: string) => void;
```

Comment:
Please use meaningful names instead of `arg0` 
It has no effect on type checking, however, it makes the code more readable.